content_scripts/hints.js

```diff
@@ -210,7 +261,25 @@ var Hints = (function(mode) {
 
     function placeHints(elements) {
         holder.attr('mode', 'click').show().html('');
-        var hintLabels = self.genLabels(elements.length);
+        var groups, hintLabels;
+        if (runtime.conf.hintGroups === true) {
+            var groups = self.genGroups(elements.length);
+            hintGroupLen = groups.groups.length;
+            switch(runtime.conf.hintGroupStart) {
+                case "middle":
+                    currentHintGroup = Math.floor(groups.groups.length / 2);
```

Comment:
`Math.ceil` seems to be better in case of that there are `groupLen + 2` hints. In which case most hints are labelled as `-1`, only the last 2 are labelled with letters.